editor/project_settings_editor.cpp

```diff
@@ -719,9 +719,18 @@ void ProjectSettingsEditor::_update_actions() {
 		item->set_range(1, action["deadzone"]);
 		item->set_custom_bg_color(1, get_color("prop_subsection", "Editor"));
 
+		const bool is_builtin_input = ProjectSettings::get_singleton()->get_input_presets().find(pi.name);
```

Comment:
Might be worth making it explicit that this is `!= NULL` (the return value of `List::find()` is an `Element` or `NULL`, in this case a `String` or `NULL`).